setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+from os import path
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+CWD = path.abspath(path.dirname(__file__))
+
+with open(path.join(CWD, "requirements.txt"), encoding="utf-8") as inp:
+    REQUIREMENTS = [el.strip() for el in inp.read().split(",")]
+
+setuptools.setup(
+    name="penn_chime",
+    version="1.0.0",
+    author="Predictive Healthcare @ Penn Medicine",
+    description="COVID-19 Hospital Impact Model for Epidemics",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/CodeForPhilly/chime",
+    project_urls={
+        "Bug Reports": "https://github.com/CodeForPhilly/chime/issues",
+        "Source": "https://github.com/CodeForPhilly/chime",
+        "Documentation": "https://codeforphilly.github.io/chime/",
+    },
+    package_dir={'': 'src'},
+    packages=setuptools.find_namespace_packages(where='src', exclude=('tests')),
+    install_requires=REQUIREMENTS,
```

Comment:
Please add the requirements here explicitly and put '-e .' in the requirements.txt file. Then the file read is no longer needed.

We'll need to test this with the deployments: `pip install .` without a requirements.txt file should be sufficient.

Reply:
I put the requirements here explicitly, I'm not sure what you mean by `-e` in the requirements file, can you give me an example?

Reply:
Thanks nick. requirments.txt does not need the list of dependencies. Replacing the contents of requirments.txt with "-e ." will cause pip to find the list in setup.py. Kinda like delegation.